### PR TITLE
Docs for AWS ECS membership scheme

### PR DESCRIPTION
**Pull request by gayashanbc**
Please don't merge
Related https://github.com/wso2/product-is/issues/6891

### CODE REVIEW

File: en/docs/setup/deployment-guide.md [hunk 1]
```diff
@@ -234,7 +235,56 @@ WSO2 supports the following membership schemes for clustering
...
             number of potential members. 
-        
+            
+        ??? tip "Click to see the instructions for AWS ECS membership scheme"  
+            !!! note
+                AWS ECS membership scheme is only supported from WUM-424342 onwards.  
+                      
+            1. First, you need a working AWS ECS Cluster.
+            for instructions on creating a cluster. Please take note of the following things when creating the cluster.
+                -   Note down the `name` and `VPC CIDR block` of the cluster as we will require them for later configurations.
```
Comment (sherenewso2) [inline en/docs/setup/deployment-guide.md]: 1. **Create a working AWS ECS Cluster. For instructions, see [add link]. Note the following when creating a cluster.**

- Note the `name` and `VPC CIDR block` of the cluster as **you** will require them **later for configurations**. 
- Ensure that the `Container instance IAM role` that you assign to the ECS cluster **has** permission to **access?** the following policy.

File: en/docs/setup/deployment-guide.md [hunk 2]
```diff
@@ -234,7 +235,56 @@ WSO2 supports the following membership schemes for clustering
...
+                    [clustering]
+                    membership_scheme = "aws-ecs"
+                    
+                    [clustering.properties]
+                    region = "us-east-1"
+                    clusterName = "ECS-IS-CLUSTER"
+                    vpcCidrBlock = "10.0.*.*"
+                    ```                    
+            Under the `clustering.properties` section, set the `region`, `clusterName`, and `vpcCidrBlock` based on 
+            the AWS ECS cluster that we crated in the previous step.
```
Comment (sherenewso2) [inline en/docs/setup/deployment-guide.md]: "that we crated" --> "that **you created**"

File: en/docs/setup/deployment-guide.md [hunk 3]
```diff
@@ -234,7 +235,56 @@ WSO2 supports the following membership schemes for clustering
...
+                    vpcCidrBlock = "10.0.*.*"
+                    ```                    
+            Under the `clustering.properties` section, set the `region`, `clusterName`, and `vpcCidrBlock` based on 
+            the AWS ECS cluster that we crated in the previous step.       
+
+            !!! note
+                Once all the configurations are complete, build a docker image including the configurations. You can 
+                consume this docker image to 
+                create a `Task Definition` and run a new `Service` or a `Task` 
+                on the `AWS ECS cluster` that we created.
```
Comment (sherenewso2) [inline en/docs/setup/deployment-guide.md]: we --> you

Comment (gayashanbc) [reply en/docs/setup/deployment-guide.md]: Resolved in https://github.com/wso2/docs-is/pull/895/commits/23568508a359cfdc535bfcdedd1782cf6b3fb7e4

File: en/docs/setup/deployment-guide.md (hunk 2 above)
Comment (gayashanbc) [reply en/docs/setup/deployment-guide.md]: Resolved in https://github.com/wso2/docs-is/pull/895/commits/23568508a359cfdc535bfcdedd1782cf6b3fb7e4

File: en/docs/setup/deployment-guide.md (hunk 1 above)
Comment (gayashanbc) [reply en/docs/setup/deployment-guide.md]: Resolved in https://github.com/wso2/docs-is/pull/895/commits/23568508a359cfdc535bfcdedd1782cf6b3fb7e4

File: en/docs/setup/deployment-guide.md [hunk 4]
```diff
@@ -277,13 +276,13 @@ WSO2 supports the following membership schemes for clustering
                     vpcCidrBlock = "10.0.*.*"
                     ```                    
             Under the `clustering.properties` section, set the `region`, `clusterName`, and `vpcCidrBlock` based on 
-            the AWS ECS cluster that we crated in the previous step.       
+            the AWS ECS cluster that you crated in the previous step.
```
Comment (sherenewso2) [inline en/docs/setup/deployment-guide.md]: Small typo - "crated" should be corrected to "created"

Comment (gayashanbc) [reply en/docs/setup/deployment-guide.md]: Resolved in https://github.com/wso2/docs-is/pull/895/commits/2fe199e6759c1b50721820f1f54cd60f0c2911b4

**Pull request merged**